joomla/joomla-cms
# Ensure calendar field displays seconds values

Pull Request for Issue #16084.

### Summary of Changes
Added switch case to capture seconds value to date helpers. Updated minified helpers.

### Testing Instructions
1. Load any form with a calendar field
2. Modify the seconds portion of the field
3. Save the form
4. Seconds field should be retained and rendered

### Expected result
Seconds field displayed with actual value (not just 00)

### Actual result
Displays as expected.

### Documentation Changes Required
NA
